slack-base/src/main/java/com/hubspot/slack/client/models/interaction/blockkit/BlockKitViewIF.java

```diff
@@ -0,0 +1,13 @@
+package com.hubspot.slack.client.models.interaction.blockkit;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@HubSpotStyle
+@JsonNaming(value = PropertyNamingStrategy.SnakeCaseStrategy.class)
+public interface BlockKitViewIF {
```

Comment:
Can we reuse https://github.com/HubSpot/slack-client/blob/master/slack-base/src/main/java/com/hubspot/slack/client/models/response/views/ViewResponseBase.java instead?